ruby-admin-portal-example/views/index.erb

```diff
@@ -14,7 +11,7 @@
                     <input type="text" placeholder='Enter Your Organization Name to Provision' id="org" name="org" class="text_input" required>
                 </div>
                 <div>
-                    <input type="text" placeholder='Enter a Space Separated List of Domains Used By the Org' id="domain" name="domain" class="text_input_2" required>
+                    <input type="text" placeholder='Enter a Space Separated List of Domains Used By the Org' id="domain" name="domain" class="text_input" required>
```

Comment:
the spacing here looks off - could you add both the text_input and text_input 2 to the second text input to eliminate the extra margin? 
![CleanShot 2022-08-12 at 12 28 12](https://user-images.githubusercontent.com/47123183/184421535-f2441b6b-b946-4ade-a41c-cb05f9769859.png)

Reply:
done!